RELEASING.md

```diff
@@ -238,16 +238,13 @@ More on that command in the section below.
    ```
 
 4. Create a PR to `master` that updates `CHANGELOG.md` files for all of the packages that
-   have just been released. The only addition to the file should be a markdown header
```

Comment:
I'd like to still keep this high level description, because there might be cases when it needs to be done manually. We can then say that it can be done automatically by the script, like you mention below

Reply:
Good point, done.